thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,48 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
```

Comment:
Can you add a test that only passes in the required variable, to ensure that's the only required one?

Also, I would suggest put a comment at the top of each "section" of this test articulating what you're testing (like "Verify that we filter correctly by both manager and name"). This is really helpful later, when tests break and you're trying to understand what the purpose of each one is.

Reply:
(Just wanted to link an [example](https://github.com/evergreen-ci/evergreen/blob/fc8c39564a88acb5b89e1badfd69c4817d1d33fb/model/project_ref_test.go#L403-L453) of what Annie means by commenting by section! Each test case has a comment explaining what the test case does)

Reply:
Thank you Minna!!